examples/niv/nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
I have never used niv. It would be nice to have a shell script, which runs niv instead of storing the generated files in the example.

Reply:
Good idea. Has been added.